frontend/components/errors/mobile_warning.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const MobileWarning = () => {
+
+    return (
+        <div className="mobile-overlay">
```

Comment:
Curious why there are two divs. Are you planning on using the style elsewhere?

Reply:
No I wasn't.... You're right, I can do it in one div. I had a modal in mind, where there is normally a background div and a forefront div. But I don't need the background in this case, thanks.